### PR TITLE
GH Actions: print draft changelog job summary

### PR DESCRIPTION
**Pull request by MetRonnie**
It's kind of useful to be able to see the rendered changelog.

https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#adding-a-job-summary

To see it, you can go to the Actions summary page https://github.com/cylc/cylc-ui/actions/runs/9082240185#summary-24958150609

image

#### Needs

- https://github.com/cylc/release-actions/pull/72

#### Check List

- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).

### CODE REVIEW

**Review by wxtim — COMMENTED** [review]
Nice - https://github.com/cylc/cylc-ui/actions/runs/9082780924

**Review by wxtim — APPROVED** [review]
Happy to have you merge, or for you to ask for a second review. 👍🏼

**Comment by MetRonnie** [comment]
1 reviewer should do. E2E test failure due to flakiness

**Pull request merged**
